docker-compose.yml

```diff
@@ -23,5 +12,20 @@ services:
     ports:
       - 5432:5432
 
+  shared:
+    image: call-home-shared
```

Comment:
What does this image + container do?

Reply:
It looks like this is build-only?

Reply:
yes the intent is to build using this step. will change to building it with an npm target

Reply:
Actually I think build only is fine. We can have both the script and this build only service